### PR TITLE
style(red buttons): change focus ring to blue

### LINKED ISSUE #6950: Change focus ring on red buttons to blue
Request from @shorlander that we use the same focus colour on all our buttons.

Current:
focus

Proposed:
Same focus ring colour on all buttons
https://design.firefox.com/photon/components/buttons.html

### PR DESCRIPTION
**Pull request by shreya99oak**
fixes #6950 

@shane-tomlinson  Pls. review

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -10166,7 +10166,7 @@
       "from": "git://github.com/vladikoff/node-uap.git#9cdd16247",
       "requires": {
         "array.prototype.find": "2.0.0",
-        "uap-core": "git://github.com/ua-parser/uap-core.git#108591a790f2a52685d694846bd43e8ce1f70adb",
```
Comment (shane-tomlinson) [inline npm-shrinkwrap.json]: Can you revert the changes to npm-shrinkwrap.json? Since you didn't update any of the dependencies, this file shouldn't need to be updated.

Comment (shreya99oak) [reply npm-shrinkwrap.json]: Yes, I'll do that!

**Comment by shreya99oak** [comment]
@shane-tomlinson Pls. review it

File: app/styles/modules/_button-row.scss [hunk 2]
```diff
@@ -58,18 +58,6 @@ button {
     background: $error-background-color;
     border: 0;
     color: $message-text-color;
-
-    &:hover {
```
[the remark this answers is not in the document]
Comment (shreya99oak) [reply app/styles/modules/_button-row.scss]: Okay,will do that!

**Comment by miralobontiu** [comment]
Verified as fixed on Stage - Train 132.
